rootfs/usr/share/opengamepadui/scripts/powertools

```diff
@@ -8,65 +8,79 @@ fi
 
 ## CPU Generics
 cpuBoost(){
-  /usr/bin/echo $1 > /sys/devices/system/cpu/cpufreq/boost
-}
-
-cpuPower(){
-  /usr/bin/cpupower $1 $2 $3
+ state=$1
+  /usr/bin/echo $state > /sys/devices/system/cpu/cpufreq/boost
 }
 
 cpuToggle(){
- /usr/bin/echo $2 > /sys/bus/cpu/devices/cpu${1}/online
+ num=$1
+ state=$2
+ /usr/bin/echo $state > /sys/bus/cpu/devices/cpu${num}/online
 }
 
 smtToggle(){
-  /usr/bin/echo $1 > /sys/devices/system/cpu/smt/control
+ state=$1
+  /usr/bin/echo $state > /sys/devices/system/cpu/smt/control
 }
 
 ## AMD APU
 amdGpuClock(){
-  /usr/bin/echo "s 0 ${1}" > /sys/class/drm/card0/device/pp_od_clk_voltage
-  /usr/bin/echo "s 1 ${2}" > /sys/class/drm/card0/device/pp_od_clk_voltage
-  /usr/bin/echo "c" > /sys/class/drm/card0/device/pp_od_clk_voltage
+  min_clk=$1
+  max_clk=$2
+  card=$3
+  /usr/bin/echo "s 0 ${min_clk}" > /sys/class/drm/${card}/device/pp_od_clk_voltage
+  /usr/bin/echo "s 1 ${max_clk}" > /sys/class/drm/${card}/device/pp_od_clk_voltage
+  /usr/bin/echo "c" > /sys/class/drm/${card}/device/pp_od_clk_voltage
 }
 
 ryzenadjLong(){
-  /usr/bin/ryzenadj $1 $2
+  cmd=$1
+  val=$2
+  /usr/bin/ryzenadj $cmd $val
 }
 
 ryzenadjShort(){
-  /usr/bin/ryzenadj $1
+  cmd=$1
+  /usr/bin/ryzenadj $cmd
 }
 
 setAmdPerfLevelValue() { # set power_dpm_force_performace_level mode
-  /usr/bin/echo $1 > /sys/class/drm/card0/device/power_dpm_force_performance_level
+  mode=$1
+  card=$2
+  /usr/bin/echo $mode > /sys/class/drm/${card}/device/power_dpm_force_performance_level
 }
 
 setAmdPerfLevelWrite() { # set power_dpm_force_performace_level write
-  /usr/bin/chmod a+w /sys/class/drm/card0/device/power_dpm_force_performance_level
+  card=$1
+  /usr/bin/chmod a+w /sys/class/drm/${card}/device/power_dpm_force_performance_level
```

Comment:
Same as above comment, we should set a default for `card` so we don't break update zip users